README.md

```diff
@@ -97,60 +97,76 @@ let encodedTrifleSignedDataProto = try trifleSignedData.serialize()
 ```kotlin
 // Check if a key already exists.
 // If no key exists, generate a public key pair
-var keyHandle =  TrifleApi.generateKeyHandle(alias: abc)
+var keyHandle: KeyHandle =  TrifleApi.generateKeyHandle("abc")
                 
-// Storing keys. Keys are codable.
-val encoder = Gson()
-let jsonKeyHandle = try encoder.toJson(keyHandle)
+// Storing keys. Key handles are serializable.
+val encoded: ByteArray = keyHandle.serialize()
 
 // Load the key from storage when we need to use it
+val decoded: KeyHandle = KeyHandle.deserialize(encoded)
 
 // Check the validity of loaded key
-// TBD trifleApi.isValid(keyHandle: keyHandle)
+val valid: Boolean = trifleApi.isValid(keyHandle)
 
 // Destroy key that is no longer in use or is invalid
-// TBD trifleApi.delete(keyHandle: keyHandle)
+trifleApi.delete(keyHandle)
         
 // Check if loaded key already has a cert. If yes, skip to checking for cert validity
 // Else if key does not have a cert OR if a new cert must be generated (eg because of existing
 // cert is already expired, or app needs to re-attest, app is re-installed, app is restored
 // from backup, ... etc)
 
 // Create cert request
-val certReq = TrifleApi.generateMobileCertificateRequest(entity = entity, keyHandle = keyHandle)
+val certReq: CertificateRequest = TrifleApi.generateMobileCertificateRequest(entity, keyHandle)
```

Comment:
can you update the doc to explain what `entity` is?

Reply:
its a value for the developer to choose. its intentionally vague since this is an open source library. https://github.com/cashapp/trifle/blob/main/android/trifle/src/main/java/app/cash/trifle/TrifleApi.kt#L21